app/helpers/needs_helper.rb

```diff
@@ -20,4 +20,22 @@ def note_category_form_id(note_category)
   def need_urgencies
     ['Urgent', 'Normal']
   end
+
+  def always_visible_needs(needs)
+    result = {}
```

Comment:
I think this can be shorted to:
```
    needs.needs_list.select do |_,value|
      ['Groceries and cooked meals', 'Physical and mental wellbeing', 'Financial support', 'Medical transport'].include? value[:name]
    end
```

Reply:
this array ` ['Groceries and cooked meals', 'Physical and mental wellbeing', 'Financial support', 'Medical transport']` can also be a constant if you use it twice